.github/ISSUE_TEMPLATE.md

```diff
@@ -1,4 +1,4 @@
 Please contribute feedback on the overall design of WebAssembly, issues about
 the textual specification, or typos from the docs section of webassembly.org.
 
-For browser issues or tooling bugs, see links at http://webassembly.org/community/feedback/.
\ No newline at end of file
+For browser issues or tooling bugs, see links at https://github.com/WebAssembly/website/blob/master/community/feedback.md.
```

Comment:
I believe the issue template renders as raw text (e.g. unformatted markdown), which makes the link syntax seem a little unnecessary if most people will have to delete it to type the body of their issue anyways.